### ISSUE TITLE
dictionary.csv is corrupted

### ISSUE THREAD

**Issue by Offpics** [anatoliykmetyuk/kanki#1]
dictionary.csv file is corrupted. I'm on Windows 10.

http://i.imgur.com/9OZMbDS.png

[All files - Dropbox link](https://www.dropbox.com/sh/ig35666gwsfp1pv/AAC9zacP9LACgQDQpzpvOXcfa?dl=0)

**Comment by MasterHowToLearn** [comment]
I believe the dictionary.csv isn't corrupted. It is just that the Korean could not be displayed properly by your computer. 
Try open a blank Excel. Go to Data, From Text, and select the csv file. Make sure to choose 65001: Unicode (UTF-8) in file origin. Also In the Step 2, in Delimiters, choose comma and tab for proper alignment.

**Comment by anatoliykmetyuk** [comment]
Yes, that's most likely the problem with the encoding. The CSV file is written in UTF-8.

Another thing to try may be to manually type some words in Korean in a text editor and save it in UTF-8 - for some systems, you need to install the support for Korean first.
